engine/src/main/java/org/camunda/bpm/engine/impl/dmn/el/ProcessEngineJuleElExpression.java

```diff
@@ -27,11 +28,13 @@
  * @author Daniel Meyer
  *
  */
-public class ProcessEngineElExpression implements ElExpression {
+public class ProcessEngineJuleElExpression implements ElExpression {
```

Comment:
Just a minor typo. We need to rename the class itself as well, I believe.
```suggestion
public class ProcessEngineJuelElExpression implements ElExpression {
```